P/PETSc/build_tarballs.jl

```diff
@@ -1,35 +1,42 @@
-using BinaryBuilder
+using BinaryBuilder, Pkg
 using Base.BinaryPlatforms
 const YGGDRASIL_DIR = "../.."
 include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
 
 name = "PETSc"
-version = v"3.18.8"
-petsc_version = v"3.18.6"
-MUMPS_COMPAT_VERSION = "5.6.2"
-SUPERLUDIST_COMPAT_VERSION = "8.1.2"   
-MPItrampoline_compat_version="5.2.1"
+version = v"3.20.5"
+petsc_version = v"3.20.5"
+SUPERLUDIST_COMPAT_VERSION = "8.2.1"   
 BLASTRAMPOLINE_COMPAT_VERSION="5.8.0"    
+SUITESPARSE_COMPAT_VERSION = "7.7"       
 
+MUMPS_COMPAT_VERSION = "5.6.2"
```

Comment:
Is it still needed?
You don't depend anymore on `MUMPS_jll.jl` or `MUMPS_seq_jll.jl`.
For information, The latest release is `5.7.3`.

Reply:
thanks for pointing this out - I have removed the compat definition where this is no longer necessary